apps/web/components/booking/pages/BookingPage.tsx

```diff
@@ -584,23 +585,27 @@ const BookingPage = ({
                     <div className="text-bookinghighlight flex items-start text-sm">
                       <FiUser
                         className={`ml-[2px] mt-[2px] inline-block h-4 w-4 ltr:mr-[10px] rtl:ml-[10px] ${
-                          booking && booking.attendees.length / eventType.seatsPerTimeSlot >= 0.5
+                          currentSlotBooking &&
+                          currentSlotBooking.attendees.length / eventType.seatsPerTimeSlot >= 0.5
```

Comment:
Why are we introducing `currentSlotBooking`? Wouldn't we have the attendee information already from the `booking` prop?

Reply:
In the case of rescheduling a booking with seats, the `booking` prop is the booking that should be rescheduled. But we also need the booking that might already exist for the new slot the user want to book (to show how many open slots this new booking has)

In [this line](https://github.com/calcom/cal.com/pull/7910/files#diff-201b6fc0fd34f1a79a852b4f445f1bae9b7ba9503e01b8826411e2e0bdeb7a5cR243) you see, if we have a reschedule uid the booking is the rescheduled booking. If it is not a reschedule then the `booking` prop will be the booking that already exists in the picked slot.